frontend/src/components/filtertable.js

```diff
@@ -160,3 +168,125 @@ export class FilterTable extends React.Component {
     );
   }
 }
+
+
+export class MetaFilter extends React.Component {
+  static propTypes = {
+    fieldOptions: PropTypes.array,  // could reference constants directly
+    runId: PropTypes.string,  // make optional?
+    setFilter: PropTypes.func,
+    customFilters: PropTypes.array, // more advanced handling of filter objects? the results-aggregator endpoint takes a string filter
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      fieldSelection: null,
+      isFieldOpen: false,
+      isValueOpen: false,
+      valueOptions: [],
+      valueSelections: [],
+    };
+  }
+
+  onFieldToggle = isExpanded => {
+    this.setState({isFieldOpen: isExpanded})
+  };
+
+  onValueToggle = isExpanded => {
+    this.setState({isValueOpen: isExpanded})
+  };
+
+  onFieldSelect = (event, selection) => {
+    this.setState(
+      // clear value state too, otherwise the old selection remains selected but is no longer visible
+      {fieldSelection: selection, isFieldOpen: false, valueSelections: [], valueOptions: [], isValueOpen: false},
+      this.updateValueOptions
+    )
+
+  };
+
+  onValueSelect = (event, selection) => {
+    // update state and call setFilter
+    const valueSelections = this.state.valueSelections;
+    let updated_values = (valueSelections.includes(selection))
+      ? valueSelections.filter(item => item !== selection)
+      : [...valueSelections, selection]
+
+    this.setState(
+      {valueSelections: updated_values},
+      () => this.props.setFilter(this.state.fieldSelection, this.state.valueSelections.join(';'))
+    )
+  };
+
+  onFieldClear = () => {
+    this.setState(
+      {fieldSelection: null, valueSelections: [], isFieldOpen: false, isValueOpen: false},
+    )
+  };
+
+  onValueClear = () => {
+    this.setState(
+      {valueSelections: [], isValueOpen: false},
+      () => this.props.setFilter(this.state.fieldSelection, this.state.valueSelections)
+    )
+  }
+
+  updateValueOptions = () => {
+    const {fieldSelection} = this.state
+    if (fieldSelection !== null) {
+      fetch(buildUrl(Settings.serverUrl + '/widget/result-aggregator',
+        {
+          group_field: fieldSelection,
+          run_id: this.props.runId,
+          additional_filters: this.props.customFilters.join(),
+        }
+      ))
+      .then(response => response.json())
+      .then(data => {
+        this.setState({valueOptions: data})
+      })
+    }
+  }
+
+  render () {
+    const {isFieldOpen, fieldSelection, isValueOpen, valueOptions, valueSelections} = this.state
+
+    return (
+      <React.Fragment>
+        <Select key="metafield_select"
```

Comment:
I'll let others weigh in. I don't have a strong opinion here, but my inclination is to not have a default.

For me and Jake exception_name would be the preferred default. For everyone else on the team it'll be assignee or component - because they want to see their own failures, not everybody else's.